### PR TITLE
Update for large disk GA

### PR DESCRIPTION
**Pull request by yuemlu**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py [hunk 1]
```diff
@@ -186,7 +186,7 @@ def load_arguments(self, _):
         c.argument('vm_name', options_list=['--vm-name'], id_part=None, completer=get_resource_name_completion_list('Microsoft.Compute/virtualMachines'))
         c.argument('new', action='store_true', help='create a new disk')
         c.argument('sku', arg_type=disk_sku, help='Underlying storage SKU')
-        c.argument('size_gb', options_list=['--size-gb', '-z'], help='size in GB. Max size: 4095 GB (certain preview disks can be larger).', type=int)
+        c.argument('size_gb', options_list=['--size-gb', '-z'], help='size in GB. Managed Disks Max size: 32767 GB, Unmanaged Disks Max size: 4095 GB', type=int)
```
Comment (adewaleo) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py]: Please don't mention unmanaged disks here. `vm disk` is for managed disks.

Rather mention unmanaged disks [here](https://github.com/Azure/azure-cli/pull/8866/files#diff-046b3d363c785b7059452b6b72566229L256) under `vm unmanaged-disk attach`.

Also nit on my part:
```suggestion
        c.argument('size_gb', options_list=['--size-gb', '-z'], help='size in GB. Managed Disks Max size: 32767 GB. Unmanaged Disks Max size: 4095 GB', type=int)
```

Comment (adewaleo) [reply src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py]: If you would like @yuemlu I can make these changes here and in #8858 in a single PR.

**Comment by yuemlu** [comment]
@adewaleo What is the next step on me?

**Comment by tjprescott** [comment]
@yuemlu why are there two separate PRs for this?

**Comment by adewaleo** [comment]
@yuemlu You didn't fully address my previous [comment](https://github.com/Azure/azure-cli/pull/8866#discussion_r268471130).

[az vm disk --size-gb](https://github.com/Azure/azure-cli/pull/8866/files#diff-046b3d363c785b7059452b6b72566229R189) should only mention "managed disks". 

On the other hand, [az vm unmanaged-disk attach --size-gb](https://github.com/Azure/azure-cli/pull/8866/files#diff-046b3d363c785b7059452b6b72566229L256) should mention "unmanaged disks".

Currently you are referring to both **managed** and **unmanaged** disks in a command that works only for managed disks.

**Comment by tjprescott** [comment]
@yuemlu please get CI passing or this PR will be closed due to inactivity.

**Comment by tjprescott** [comment]
Closing due to inactivity. Feel free to address comments and reopen.

**Pull request closed without merging**